pallets/staking/src/lib.rs

```diff
@@ -269,6 +273,11 @@ pub mod pallet {
 	pub type Nonce<T: Config> =
 		StorageMap<_, Blake2_128Concat, DefaultVaultId<T>, T::Index, ValueQuery>;
 
+	// store with all the reward currencies in use
+	#[pallet::storage]
+	pub type RewardCurrencies<T: Config> =
```

Comment:
What I mean is: doesn't the `RewardCurrencies` in the pooled-rewards pallet already keep track of this?

Reply:
Yes it does, it has it by pool id, but it does. It was the other alternative in fact. The rationale behind this change was to have more control of the currencies that are going to be distributed. For example, in case in the future we wanted to change/remove some of them, and also the fact that this new store does not differentiate between pools.

Reply:
Okay, I did not look properly and only realize now that here the currencies are not accounted by pool id but in general, so that it is indeed a difference.

So then it makes sense.